neo/Consensus/RecoveryMessage.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.IO;
+using Neo.IO;
+using Neo.Ledger;
+
+namespace Neo.Consensus
+{
+    internal class RecoveryMessage : PrepareRequest
```

Comment:
Maybe we can change this type to:

```c#
internal class RecoveryMessage : ConsensusMessage
{
    public PrepareRequest PrepareRequest;
    public byte[][] PrepareMsgWitnessInvocationScripts;
    public uint PrepareRequestPayloadTimestamp;
    public byte[][] ChangeViewWitnessInvocationScripts;
}
```

Reply:
We could do that, but what is the advantage? The RecoveryMessage's `ViewNumber` would be redundant with the `PrepareRequest` view number.

Reply:
Sounds precise, Erik. The timestamp was the trick when changed from the Payload to the Message.

Reply:
I have to do something different since I may need to create the recovery message now without the `PrepareRequest` and right now `PrepareRequest` requires the `MinerTransaction` to be present. I'm fixing this.

Reply:
I thought of a way to handle it and still extend `PrepareRequest` by just changing the way `RecoveryMessage` is serialized / deserialized.